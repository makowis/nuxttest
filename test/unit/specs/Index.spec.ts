import { mount } from '@vue/test-utils'
import Index from '@/pages/index.vue';

describe('index.vue', () => {
  const vm = mount(Index).vm;

  it('should match the snapshot', () => {
    expect(vm.$el).toMatchSnapshot();
  });
});
