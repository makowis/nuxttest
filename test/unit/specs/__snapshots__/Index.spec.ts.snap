// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.vue should match the snapshot 1`] = `
<div
  class="Top"
>
  <tbt-h-1>
    <tbt-blink>
      定時後ビールタイム公式サイト
    </tbt-blink>
  </tbt-h-1>
   
  <p>
    <tbt-blink>
      🎉🎉🎉🎉🎉🎉🎉M3-2020春参戦決定！🎉🎉🎉🎉🎉🎉
    </tbt-blink>
  </p>
   
  <tbt-marquee>
    
    M3-2020春に参戦！パワーアップしたサウンドをお届けします！
  
  </tbt-marquee>
   
  <tbt-access-counter-section
    access="0"
    realtime="0"
  />
   
  <tbt-news-section />
   
  <tbt-circle-cut-section />
   
  <tbt-circle-space-infosection />
   
  <tbt-mail-section />
   
  <tbt-menu-section />
</div>
`;
